h/templates/activity/search.html.jinja2

```diff
@@ -348,18 +348,26 @@
     {% endif %}
 
     {% if timeframes %}
-      <ol class="search-result-list {%- if more_info %} search-result-hide-on-small-screens{% endif %}">
-        {% for timeframe in timeframes %}
+
+      <div class="search-results">
+
+        {% if q or user or group %}
+        <div class="search-results__total"><b>{% trans %}Annotations{% endtrans %}</b> {{total}}</div>
```

Comment:
Is the `<b>` here purely for styling? If so, I'd suggest moving that to `font-weight: bold` in `search-results__total`.